src/Avalonia.Base/Input/DataFormat.cs

```diff
@@ -32,6 +33,12 @@ private protected DataFormat(DataFormatKind kind, string identifier)
     /// </summary>
     public static DataFormat<string> Text { get; } = CreateUniversalFormat<string>("Text");
 
+    /// <summary>
+    /// Gets a data format representing a bitmap.
+    /// Its data type is <see cref="Bitmap"/>.
+    /// </summary>
+    public static DataFormat<Bitmap> Image { get; } = CreateUniversalFormat<Bitmap>("Image");
```

Comment:
Maybe 
```suggestion
    public static DataFormat<Bitmap> Bitmap { get; } = CreateUniversalFormat<Bitmap>("Bitmap");
```

Also matches https://learn.microsoft.com/en-us/dotnet/api/system.windows.dataformats.bitmap?view=windowsdesktop-9.0#system-windows-dataformats-bitmap